test/unit/helpers/xhr/MFA_CHALLENGE_multipleWebauthn.js

```diff
@@ -0,0 +1,92 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2016-08-17T02:29:38.000Z",
+    "status": "MFA_CHALLENGE",
+    "_embedded": {
+      "challenge": {
+        "nonce": "someNonce",
+        "timeoutSeconds": 20
+      },
+      "user": {
+        "id": "00ukjpMDb9O7CjJbx0g3",
+        "passwordChanged": "2016-08-15T18:31:57.000Z",
+        "profile": {
+          "login": "exampleUser@example.com",
+          "firstName": "Test",
+          "lastName": "User",
+          "locale": "en_US",
+          "timeZone": "America/Los_Angeles"
+        }
+      },
+      "challenge": {challenge: "kygOUtSWURMv_t_Gj71Y", extensions: {"appid": "https://foo.com"}},
+      "factors": [{
+        "id": "webauthnFactorId1",
+        "factorType": "webauthn",
+        "provider": "FIDO",
+        "vendorName": "FIDO",
+        "profile": {"credentialId": "vdCxImCygaKmXS3S_2WwgqF1LLZ4i_2MKYfAbrNByJOOmSyRD_STj6VfhLQsLdLrIdgvdP5EmO1n9Tuw5BawZw"},
+        "_embedded": {
+          "challenge": {
+            "challenge": "abcd",
```

Comment:
nit: `challenge1` and rest the same, similar to your ids `challenge2` and `challenge3`